packages/ubuntu_desktop_installer/lib/pages/keyboard_layout/keyboard_layout_model.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/foundation.dart';
+import 'package:keyboard_info/keyboard_info.dart';
+import 'package:subiquity_client/subiquity_client.dart';
+
+import '../../keyboard_model.dart';
+
+/// Implements the business logic of the Keyboard Layout page.
+class KeyboardLayoutModel extends ChangeNotifier {
+  /// Creates a model with the specified [client] and [keyboardModel].
+  KeyboardLayoutModel({
+    required SubiquityClient client,
+    required KeyboardModel keyboardModel,
```

Comment:
would be nice to rename KeyboardModel to avoid confusion between similarly named models.

Some ideas:
- KeyboardDatabase
- KeyboardRepository
- KeyboardService

Reply:
Agreed. `KeyboardService` sounds fine to me.

Reply:
:+1: I'll push a separate PR later